lib/helpers.rb

```diff
@@ -490,9 +507,23 @@ def chart_data? data, type
       type_data.any?
     end
 
-    def process_chart_data data, type
-      type_data = data.select{|hash| hash["id"] == type }
-      type_data[0].fetch("yearMonths",[]) if type_data.any?
+    def process_chart_data data, types, yop
+      type_data = []
+      types.each do |tpy|
+        type_data = data.select{|hash| hash["id"] == tpy }
+      end
+      if type_data.any?
+        # if more than 10 years are to be shown
+        if type_data[0]["yearMonths"].size > 100 
```

Comment:
@kjgarza why `100` and not `120` for 10 years?

Reply:
@kjgarza have you looked into the effort needed to do the filtering by time range in the `lagottino` API?

Reply:
yes; I don't think is a good idea to filter in laggotino or sashimi, you would need an extra API call for each DOI to get the yop.

I reduced to 100 due to spacing. so we need around 960 px for 120 bars (7px). And on a medium size screen that won't fit the chart inside the box with labels and padding. So i took out 20 bars

Reply:
Ok, doing this in `lagottino` sounds like more work. But we have the `publicationYear` in the `subj` and `obj` metadata.

What is the minimal bar size that looks good? Maybe you get away with 6px including space. In the current implementation you would show a maximum of 8 years and 4 months, which will be confusing.

Reply:
fix 89baad49981f685e9a80612ac5b3065b5e7539c5